### PR TITLE
feat: show all articles of latest issue

### LINKED ISSUE #267: feat: show all articles of latest issue
Currently only 11 articles are showing in landing page.
Modify the code in such a way so that all the articles of latest issue show in the landing page.

### PR DESCRIPTION
**Pull request by anish-patro**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #267 

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/screens/Home.js [hunk 1]
```diff
@@ -14,6 +15,8 @@ import Trending from '../components/homepage/Trending';
 import ArticleGrid from '../components/widgets/article/ArticleGrid';
 
 function Home({ issues, squiggles }) {
+  const mobMatches = useMediaQuery(theme.breakpoints.down('md'));
+  const tabletMatches = useMediaQuery(theme.breakpoints.down('lg'));
```
Comment (120EE0692) [inline client/src/screens/Home.js:19]: @anish-patro mobMatches is not used. Try to solve this issue by using between.

File: client/src/screens/Home.js [hunk 2]
```diff
@@ -32,22 +35,44 @@ function Home({ issues, squiggles }) {
...
+            />
+            {articles.length > 4 && (
+              <ArticleCardStack articleList={articles.slice(4, 8)} />
+            )}
+            {articles.length > 8 && (
+              <ArticleCardStack
+                articleList={articles.slice(8, articles.length)}
+              />
+            )}
+          </>
```
Comment (120EE0692) [inline client/src/screens/Home.js:75]: @anish-patro I think this can be done in a better way.

example: 
matches ? 3 : 4

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

**Review by 120EE0692 — APPROVED** [review]

File: client/src/screens/Home.js [hunk 3]
```diff
@@ -14,6 +15,7 @@ import Trending from '../components/homepage/Trending';
 import ArticleGrid from '../components/widgets/article/ArticleGrid';
 
 function Home({ issues, squiggles }) {
+  const tabletMatches = useMediaQuery(theme.breakpoints.between('sm', 'lg'));
```
Comment (rutajdash) [inline client/src/screens/Home.js:18]: Why do we need to consider a separate value for tablet view at all? The UI adapts fine the same doesn't it?

Comment (anish-patro) [reply client/src/screens/Home.js:18]: If we will not use that then it  will look like this in tablet/small laptops view
Screenshot (88)

**Comment by rutajdash** [comment]
@anish-patro resolve all the conflicts please.

**Pull request merged**